pkg/skaffold/build/custom/dependencies.go

```diff
@@ -53,3 +52,8 @@ func GetDependencies(ctx context.Context, workspace string, a *latest.CustomArti
 		return list.Files(workspace, a.Dependencies.Paths, a.Dependencies.Ignore)
 	}
 }
+
+func getDockerBuildConfig(ws string, artifact string, a *latest.CustomArtifact) docker.BuildConfig {
+	dockerfile := a.Dependencies.Dockerfile
```

Comment:
should we normalize the Dockerfile path here itself instead of later?

Reply:
That is a good suggestion. So i `normalizeDockerpath` is used a lot many places. while going through the code, there is a huge scope of avoiding repetition across `docker.SyncMap` and `docker.GetDependencies`

I don't want to error out here in this stage of creating a `BuildConfig`